tools/cjxl_main.cc

```diff
@@ -597,7 +598,13 @@ jxl::Status GetPixeldata(const std::vector<uint8_t>& image_data,
       return jxl::extras::Codec::kJPG;
     }
 #endif
-    // TODO(tfish): Bring back EXR and PSD.
+#if JPEGXL_ENABLE_EXR
+    if (jxl::extras::DecodeImageEXR(encoded, color_hints, size_constraints,
+                                    &ppf)) {
+      return jxl::extras::Codec::kJPG;
```

Comment:
Shouldn't this be something else?

Reply:
Yup... Although it seems completely ignored xD